omnibus/files/private-chef-cookbooks/private-chef/recipes/rabbitmq.rb

```diff
@@ -183,13 +183,24 @@
     retries 10
   end
 
-  execute "#{rmq_plugins} enable rabbitmq_management" do
-    environment (rabbitmq_env)
-    user opc_username
-    not_if "#{rmq_plugins} list | grep rabbitmq_management | grep -v rabbit_management_agent | grep E"
-    # management plugin needs a rabbit restart
-    notifies :restart, 'runit_service[rabbitmq]', :delayed
-    retries 10
+  rabbitmq_management_status = "#{rmq_plugins} list | grep rabbitmq_management  | grep -v rabbitmq_management_agent | grep -v rabbitmq_management_visualiser"
+  if rabbitmq['management_enabled']
+    execute "#{rmq_plugins} enable rabbitmq_management" do
+      environment (rabbitmq_env)
+      user opc_username
+      not_if "#{rabbitmq_management_status} | grep E"
+      # management plugin needs a rabbit restart
+      notifies :restart, 'runit_service[rabbitmq]', :delayed
+      retries 10
+    end
+  else
+    execute "#{rmq_plugins} disable rabbitmq_management" do
+      environment (rabbitmq_env)
+      user opc_username
+      notifies :restart, 'runit_service[rabbitmq]', :delayed
+      not_if "#{rabbitmq_management_status} | grep -v E"
```

Comment:
No, I mean not_if.  What that block is doing is ensuring we only try to change the state of the addons when they aren't already in the desired state.

I could have used if without the -v, but -v looks for lines without the pattern.